_opensearch/rest-api/rank-eval.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: default
+title: Ranking evaluation
+parent: REST API reference
+nav_order: 82
+---
+
+# Ranking evaluation
+
+The [rank]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/rank/) eval endpoint allows you to evaluate the quality of ranked search results.
+
+## Path and HTTP methods
+
+```
+GET <index_name>/_rank_eval 
+POST <index_name>/_rank_eval
+```
+
+## Query parameters
+
+Query parameters are optional.
+
+Parameter | Data Type | Description
+:--- | :---  | :---
+ignore_unavailable | boolean | Defaults to `false`. When set to `false` the response body will return an error if an index is closed or missing.
+allow_no_indices | boolean | Defaults to `true`. When set to `false` the response body will return an error if a wildcard expression points to an indices that are closed or missing.
+expand_wildcards | string | Expand wildcard expressions for indices that are `open`, `closed`, `hidden`, `none`, or `all`.
+search_type | enum | Set search type.
+
+## Request fields
+
+The request body must contain at least one parameter.
+
+Field Type | Description
+:--- | :---  
+id | Document or template ID.
+requests | Set multiple search requests within the request field section.
+ratings | Document relevance score.
+k | The number of documents returned per query. Default is set to 10.
+relevant_rating_threshold | The threshold that documents are considered relevant. Default is set to 1.
```

Comment:
The description needs a little revision. Do we mean "The threshold at which documents are considered relevant"?